src/sysl/core/syslloader.py

```diff
@@ -309,6 +305,14 @@ def infer_expr_type(expr, top=True):
         infer_expr_type(v.expr)
 
 
+def infer(module):
```

Comment:
I needed this a public function to create true unit tests without accessing input files.
I'm mot sure if this is the most suited function name - I'm happy to take suggestions.

Reply:
`postprocess` should do.

Reply:
Done